patches/net/minecraft/world/level/ServerExplosion.java.patch

```diff
@@ -1,12 +1,21 @@
 --- a/net/minecraft/world/level/ServerExplosion.java
 +++ b/net/minecraft/world/level/ServerExplosion.java
+@@ -165,7 +_,7 @@
+         return new ObjectArrayList<>(set);
+     }
+ 
+-    private void hurtEntities() {
++    private void hurtEntities(List<BlockPos> blocks) {
```

Comment:
The vanilla method needs to be kept, marked as deprecated and forward to the new method with a default value (i.e. empty list in this case).